### PR TITLE
Installation

### PR DESCRIPTION
**Pull request by mattias-p**
Updated installation instruction for FreeBSD.
Installation now uses `cpan -i` instead of `git clone`.
The web app files are now installed in the Zonemaster-GUI dist directory.

The instructions for CentOS, Debian, Docker and Ubuntu also need to be tested and updated.

### CODE REVIEW

File: Makefile.PL [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
-    NAME                => 'Zonemaster::GUI',
-    AUTHOR              => q{Michal TOMA <toma@nic.fr>},
-    VERSION_FROM        => 'lib/Zonemaster/GUI.pm',
-    ABSTRACT            => 'The Zonemaster project Web Frontend',
-    ($ExtUtils::MakeMaker::VERSION >= 6.3002
-      ? ('LICENSE'=> 'perl')
-      : ()),
-    PL_FILES            => {},
+name 'Zonemaster-GUI';
+abstract 'The Zonemaster project Web Frontend';
```
Comment (matsduf) [inline Makefile.PL]: "Web frontend for Zonemaster"

File: docs/Installation.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+    sudo mkdir -p /usr/share/doc/zonemaster
+    sudo cp -a zm_app /usr/share/doc/zonemaster
+
+6) Start the server:
+
+    sudo starman --listen=:80 /usr/share/doc/zonemaster/zm_app/bin/app.pl
+
+The Doc directory in the source code also has an example Upstart file for the Web GUI starman server.
+
```
Comment (matsduf) [inline docs/Installation.md]: Update with cpan installation.

File: docs/Installation.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+	docker run -t -p 50080:50080 afniclabs/zonemaster-gui
+	
+Use the Zonemaster GUI by pointing your browser at
+
+	http://localhost:50080/
+	
+Use the Zonemaster engine from the command line
+
+	docker run -t -i afniclabs/zonemaster-gui bash
+
```
Comment (matsduf) [inline docs/Installation.md]: Update with cpan installation.

Comment (mattias-p) [reply docs/Installation.md]: Can you provide a wording that I can use?

File: Makefile.PL (hunk 1 above)
Comment (mattias-p) [reply Makefile.PL]: Done.

**Pull request merged**